cmake/scripts/windows/ArchSetup.cmake

```diff
@@ -32,9 +32,11 @@ set(DEPS_FOLDER_RELATIVE project/BuildDependencies)
 set(DEPENDENCIES_DIR ${CMAKE_SOURCE_DIR}/${DEPS_FOLDER_RELATIVE}/${ARCH})
 set(MINGW_LIBS_DIR ${CMAKE_SOURCE_DIR}/${DEPS_FOLDER_RELATIVE}/mingwlibs/${ARCH})
 
+# mingw libs
 list(APPEND CMAKE_PREFIX_PATH ${MINGW_LIBS_DIR})
 list(APPEND CMAKE_LIBRARY_PATH ${MINGW_LIBS_DIR}/bin)
-list(APPEND CMAKE_PREFIX_PATH ${DEPENDENCIES_DIR})
+# dependencies
+list(PREPEND CMAKE_PREFIX_PATH ${DEPENDENCIES_DIR})
```

Comment:
@Karlson2k `PREPEND` list command was introduced in cmake 3.15 we have to change requirement of cmake minimal version https://github.com/xbmc/xbmc/blob/master/CMakeLists.txt#L1